docs/modules/getting-started/pages/persistence.adoc

```diff
@@ -0,0 +1,192 @@
++
+[source,shell]
+----
+docker run --network hazelcast-network -it --rm hazelcast/hazelcast:5.0 hz-cli --targets hello-world@<IP address of one of the members> sql
+----
+. Create the map.
++
+[source,shell]
+----
+CREATE MAPPING mydistributedmap TYPE IMap OPTIONS ('keyFormat'='varchar','valueFormat'='varchar');
+----
+. Add data to the map.
++
+[source,shell]
+----
+SINK INTO mydistributedmap VALUES
+('1', 'John'),
+('2', 'Mary'),
+('3', 'Jane');
+----
+. Exit the SQL shell using the `exit` command.
+
+== Step 4. Monitor the Map
+
+Now, check the map in Management Center.
+
+. Open a new terminal and start Management Center.
++
+[source,shell]
+----
+docker run \
+    --network hazelcast-network \
+    -p 8080:8080 hazelcast/management-center:latest-snapshot
+----
+. In a web browser, go to localhost:8080 and enable Dev Mode.
++
+image:mc-dev-mode.png[Enabling dev mode in Management center]
+. You will see a **Connect** box on the screen; click on it and enter your cluster's name (`hello-world`) and IP addresses/ports of three members.
++
+image:connect-cluster.png[Connecting Management Center to the cluster]
+. Once you click on the **Connect** button, you should see that the cluster is in an active state and has three members.
++
+image:cluster-connected.png[Management Center is now connected to the cluster]
+. Click on **View Cluster** and go to **Storage > Maps**. You can confirm that the map you've created in Step 3 has data with three entries.
++
+image:cluster-maps.png[Map listing]
+. As an optional step, if you want to see the details of `mydistributedmap`, click on it on the screen shown above and check the "Map Statistics" box.
++
+image:map-details.png[Map details]
+
+== Step 5. Shut Down the Cluster
+
+Now, you'll shut down the whole cluster using Management Center.
+
+. While in Management Center, go to **Cluster** > **Administration**, and select the **Cluster State** tab. 
++
+image:cluster-state.png[Shutting down the cluster]
+. Click on the **Shutdown** button and confirm it on the dialog shown afterwards.
+
+Management Center now shows that it is disconnected from the cluster. You can also confirm
+this by checking the terminals where you started the members; they are now exited to the shell, meaning all the members are gone.
+
+== Step 6. Restart the Cluster
+
+Restart the cluster by starting all the members; run the commands in Step 1 and Step 2 above.
+
+== Step 7. Check the Map Data
+
+Once all the members are started, go to Management Center, and you can see that it reconnects to the cluster.
+Check your map as instructed in Step 4 above; you will see the map and its data has been recovered.
+If the persistence was not enabled, the data would be lost in case of a cluster shutdown. 
+
+== Step 8. Shut Down the Cluster
+
+Shut down the cluster you've created in this tutorial so that you can start a fresh one when you
```

Comment:
I would also add some information about deleting the `persist` directory that was used to persist data on the host device.